mira/examples/sir.py

```diff
@@ -109,3 +110,25 @@
               {"tanvar": "R'"}],
      "Wn": [{"efflux": 1, "effusion": 1},
             {"efflux": 2, "effusion": 2}]}
+
+
+infection_symptomatic = GroupedControlledConversion(
+    subject=susceptible,
+    outcome=infected_symptomatic,
+    controllers=[infected_symptomatic, infected_asymptomatic],
+)
+infection_asymptomatic = GroupedControlledConversion(
+    subject=susceptible,
+    outcome=infected_asymptomatic,
+    controllers=[infected_symptomatic, infected_asymptomatic],
+)
+recovery = NaturalConversion(
+    subject=infected,
+    outcome=recovered,
+)
+svir = TemplateModel(
```

Comment:
@bgyori so import this in the notebook, do a visualization of the "before", then show that you can add appropriate new templtaes, then do a visualization of "after". Then you can potentially do comparisons to the other models that include "hospitalization" mentioned in this notebook as well and the other ones in the scenario book